src/bijectors/truncated.jl

```diff
@@ -68,6 +68,25 @@ end
 
 with_logabsdet_jacobian(b::TruncatedBijector, x) = transform(b, x), logabsdetjac(b, x)
 
+function truncated_inv_logabsdetjac(y, a, b)
+    lowerbounded, upperbounded = isfinite(a), isfinite(b)
+    if lowerbounded && upperbounded
+        abs_y = abs(y)
+        return log(b - a) - abs_y + 2 * LogExpFunctions.log1pexp(-abs_y)
+    elseif lowerbounded || upperbounded
+        return convert(promote_type(typeof(y), typeof(a), typeof(b)), y)
+    else
+        return zero(y)
```

Comment:
This branch could still lead to a type instability. Instead of the `promote_type` etc. in the second branch it might be easier to just add a
```julia
y, a, b = promote(y, a, b)
```
at the top of the function body (outside of `if lowerbounded && ...`).